8/fpm/drupal_entrypoint.sh

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+
+set -e
+
+#Check for drupal version file
+if ! [ -e /var/www/html/drupal_version.txt ]; then
+  #Doesn't exist - initialize one
+  touch /var/www/html/drupal_version.txt
```

Comment:
Wouldn't it be better to skip this process of storing version in a txt file and then just proceed installing this specific version? Personally I don't see the purpose of having a file with version name when you already have a variable that has the version.